lib/node_modules/@stdlib/plot/table/unicode/README.md

```diff
@@ -0,0 +1,407 @@
+
+#### UnicodeTable.prototype.maxOutputWidth
+
+Maximum output width. If the output is longer than the maximum output width, it is wrapped into multiple tables.
+
+```javascript
+var table = new UnicodeTable();
+
+// Set:
+table.maxOutputWidth = 200;
+
+// Get:
+var width = table.maxOutputWidth;
+// returns 200
+```
+
+<a name="prop-row-separator"></a>
+
+#### UnicodeTable.prototype.rowSeparator
+
+Row Separator character. Must be a character string or `'None'`. A value of `'None'` indicates that rows won't be separated.
+
+```javascript
+var table = new UnicodeTable();
+
+// Set:
+table.rowSeparator = '-';
+
+// Get:
+var separator = table.rowSeparator;
+// returns '-'
+```
+
+* * *
+
+### Methods
+
+<a name="method-add-row"></a>
+
+#### UnicodeTable.prototype.addRow( row )
+
+Adds a row to table data. The row must have the same number of columns as existing data. 
+
+```javascript
+var table = new UnicodeTable( [ [ 1, 2, 3 ], [ 4, 5, 6 ] ] );
+
+var data = table.getData().data;
+// returns [ [ 1, 2, 3 ], [ 4, 5, 6 ] ]
+
+table.addRow( [ 7, 8, 9 ] );
+
+data = table.getData().data;
+// returns [ [ 1, 2, 3 ], [ 4, 5, 6 ], [ 7, 8, 9 ] ]
+```
+
+<a name="method-get-data"></a>
+
+#### UnicodeTable.prototype.getData()
+
+Gets table data and headers.
+
+```javascript
+var table = new UnicodeTable( [ [ 1, 2, 3 ], [ 4, 5, 6 ] ], [ 'col1', 'col2', 'col3' ] );
+
+var data = table.getData();
+// returns { 'data': [ [ 1, 2, 3 ], [ 4, 5, 6 ] ], 'headers': [ 'col1', 'col2', 'col3' ] }
+```
+
+<a name="method-render"></a>
+
+#### UnicodeTable.prototype.render()
+
+Renders a Unicode table.
+
+```javascript
+var table = new UnicodeTable( [ [ 1, 2, 3 ], [ 4, 5, 6 ] ], [ 'col1', 'col2', 'col3' ] );
+
+var str = table.render();
+// returns '...'
+```
+
+<a name="method-set-data"></a>
+
+#### UnicodeTable.prototype.setData()
+
+Sets table data and headers.
+
+```javascript
+var table = new UnicodeTable();
+
+table.setData( [ [ 1, 2, 3 ], [ 4, 5, 6 ] ], [ 'col1', 'col2', 'col3' ] );
+
+var data = table.getData();
+// returns { 'data': [ [ 1, 2, 3 ], [ 4, 5, 6 ] ], 'headers': [ 'col1', 'col2', 'col3' ] }
+```
+
+* * *
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<section class="examples">
+
+* * *
+
+## Examples
+
+<!-- eslint-disable stdlib/no-redeclare -->
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var randu = require( '@stdlib/random/base/randu' );
+var Float64Array = require( '@stdlib/array/float64' );
+var array = require( '@stdlib/ndarray/array' );
+var ctor = require( '@stdlib/plot/table/unicode' );
+
+var headers;
+var table;
+var data;
+var str;
+var i;
+
+// Generate some random data...
+data = new Float64Array( 50 );
+for ( i = 0; i < data.length; i++ ) {
+    data[ i ] = randu() * 100.0;
+}
+data = array( data, {
+    'shape': [ 10, 5 ]
+});
+
+// Generate headers...
+headers = new Float64Array( 5 );
+for ( i = 0; i < headers.length; i++ ) {
+    headers[ i ] = randu() * 100.0;
+}
```

Comment:
I suggest just hardcoding a list of strings here. Generating random headers is unlikely in user code.